web/asgi.py

```
@@ -14,35 +14,35 @@
-# Initialize Django settings
+# Set Django settings module before any Django imports
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "web.settings")
+
+# Initialize Django before importing anything that requires ORM
 django.setup()
 
-try:  # Import websocket URL patterns (must exist when Channels is assumed installed)
-    from web.routing import websocket_urlpatterns  # type: ignore
-except Exception:  # pragma: no cover - fail safe with empty list
-    websocket_urlpatterns = []  # type: ignore
+from typing import Any, Awaitable, Callable, Dict  # noqa: E402
```

Ruff (0.14.1)

Import from `collections.abc` instead: `Awaitable`, `Callable`

Import from `collections.abc`

(UP035)